src/errors/custom.error.ts

```diff
@@ -24,3 +18,19 @@ export abstract class CustomError extends Error {
         }
     }
 }
+
+
```

Comment:
My thought was that if I create a Word utility, wrapping the Word APIs, wouldn't I want this same type for it as well?

But I don't feel strongly about it, I can move it back.  I just have a feeling I'll be moving it forward once we have a Word helper...